roles/downstream-setup/tasks/yum_repos.yml

```diff
@@ -0,0 +1,11 @@
+---
+- fail:
+    msg: "Please define the repos to install with the var yum_repos."
+  when: yum_repos|length == 0
+
+- name: Download yum repos to /etc/yum.repos.d
+  get_url:
+    url: "{{ item }}"
+    dest: "/etc/yum.repos.d/{{ item }}.repo"
```

Comment:
This is bugged now because we changed item to be a URL to a .repo file.  If we want to the flexibility of giving a full URL here we'll also need to provide the name of the repo file to create.  Unless we want to try parsing the url, which is an option but that might get hairy.